client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/HttpConnection.spec.ts

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-import { IHttpClient } from "../Microsoft.AspNetCore.SignalR.Client.TS/HttpClient"
+import { TestHttpClient } from "./TestHttpClient"
```

Comment:
Super nit: Do we in general not care about the order of imports? Not a big deal

Reply:
We should order them. It's just harder because we don't have a typescript auto-formatter to help us :)

Reply:
Also, I'm not sure what we should order by here. The `from` clause? Or the list of imported symbols. It's worth thinking about, but we're pretty rough with it throughout the code right now.